Source/ASButtonNode.mm

```diff
@@ -197,13 +197,11 @@ - (void)updateTitle
     newTitle = _normalAttributedTitle;
   }
 
-  ASTextNode *titleNode = _titleNode;
-  
-  if ((titleNode != nil || newTitle.length > 0) && [titleNode.attributedText isEqualToAttributedString:newTitle] == NO) {
-    titleNode.attributedText = newTitle;
+  if ((_titleNode != nil || newTitle.length > 0) && [self.titleNode.attributedText isEqualToAttributedString:newTitle] == NO) {
```

Comment:
@nguyenhuy Can you add a comment above this line *why* we call `self.titleNode` explicitly. Thanks!